vendor/assets/javascripts/jquery.textile.js

```diff
@@ -69,14 +69,28 @@
 
       // move textarea to container
       $('.textile-inner', this.options.$wrap).append(this.$element);
-      this.$element.css('resize', 'none');
-      this.$element.css('width', '100%');
-      this.$element.attr('rows', 20);
+      this.$element.addClass('h-100').attr('rows', 20).wrap('<div class="col-6"></div>');
 
-      // add Preview to container and hide
+      // add Preview to container and load
       this.options.$preview = $(this.options.tpl.preview);
       $('.textile-inner', this.options.$wrap).append(this.options.$preview);
-      this.options.$preview.hide();
+      this.options.$preview.wrap('<div class="col-6"></div>');
+      this._loadPreview();
+
+      // Sync preview
+      var typingTimer;
+      var doneTypingInterval = 800;
```

Comment:
This same feature in the plugin manager uses 500ms as a buffer. It might be good to be consistent.

Reply:
I'll update it for consistency. I couldn't find it in the plugin manager though, have a link?

Reply:
@aapomm pinged this on slack: https://github.com/secroots/dradis-pro/blob/develop/app/assets/javascripts/mintcreek/modules/admin/plugins.js.coffee#L12

I'm going to update it to match the `textchange` we use, and remove the onkeydown event I added just for consistency.